opentelemetry-distro/tests/test_configurator.py

```diff
@@ -143,3 +147,16 @@ def test_trace_init_custom_id_generator(self, mock_iter_entry_points):
         _init_tracing({}, id_generator)
         provider = self.set_provider_mock.call_args[0][0]
         self.assertIsInstance(provider.id_generator, CustomIdGenerator)
+
+
+class TestExporterNames(TestCase):
+    def test_otlp_exporter_overwrite(self):
+        for exporter in [EXPORTER_OTLP, EXPORTER_OTLP_SPAN]:
+            environ[OTEL_TRACES_EXPORTER] = exporter
+            self.assertEqual(_get_exporter_names(), [EXPORTER_OTLP_SPAN])
+            del environ[OTEL_TRACES_EXPORTER]
+
+    def test_multiple_exporters(self):
+        environ[OTEL_TRACES_EXPORTER] = "jaeger,zipkin"
+        self.assertEqual(sorted(_get_exporter_names()), ["jaeger", "zipkin"])
+        del environ[OTEL_TRACES_EXPORTER]
```

Comment:
Thanks. You could simplify this by wrapping `@patch.dict(environ, {OTEL_TRACES_EXPORTER: "jaeger,zipkin"})` the test method.